apps/heureka/src/components/Services/useFetchServiceImageVersionIssues.ts

```diff
@@ -66,7 +69,7 @@ export const useFetchServiceImageVersionIssues = ({
         },
         fetchPolicy: "network-only",
       }),
-    []
+    [pageSize, searchTerm]
```

Comment:
Page size doesn't change. This attribute can be also added to the static attributes in the `useGetServiceImageVersionIssuesLazyQuery` and save one dependency here.